### PR TITLE
Add Cypress 4390x board apps and build config

### PR DESCRIPTION
**Pull request by CypressHJ**
Add cypress 4390x board example apps and build config files. 

application : apps/examples/cy4390x_demo
config files : build/configs/cy4390x

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/cy4390x_demo/Makefile [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+else
+  INSTALL_DIR = $(BIN_DIR)
+endif
+
+CONFIG_EXAMPLES_CY4390x_DEMO_PROGNAME ?= cy4390x_demo$(EXEEXT)
+PROGNAME = $(CONFIG_EXAMPLES_CY4390x_DEMO_PROGNAME)
+
+ROOTDEPPATH = --dep-path .
+
+CFLAGS+=-I$(TOPDIR)/../external/WICED/bcm4390x/include
```
Comment (pradeepns85) [inline apps/examples/cy4390x_demo/Makefile]: Where are these files ?

File: apps/examples/cy4390x_demo/cy4390x_demo_main.c [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+#include <stdio.h>
+#include <stdint.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <string.h>
+#include <sys/types.h>
+#include <tinyara/config.h>
+#include <apps/shell/tash.h>
+
+#include <wiced.h>
```
Comment (pradeepns85) [inline apps/examples/cy4390x_demo/cy4390x_demo_main.c]: where is this file ?

**Comment by CypressHJ** [comment]
Hi girishks2000, pradeepns85. 

this "Pull requests" is a part of cypress 4390x board BSP code and, it's created by request of Samsung Things Platform Lab(SR). please contact with seongil.hahm@samsung.com or pilseob.kang@samsung.com or sangwon03.an@samsung.com
thanks.  

BR, HJ Lee.

**Comment by CypressHJ** [comment]
As in today discussion with Samsung Things Platform Lab(SR), this pull request will be closed. 
If we are going to fix copyright header, then, create another pull request again. thanks. 

BR,  HJ Lee.

**Pull request closed without merging**